docs/Bare-Metal/ZFS.md

```diff
@@ -344,5 +345,26 @@ proc4 2 187 1724229556
 proc4ops 72 0 0 0 283322 277380 92 2950 0 7998 2976415 159260 147 1471402 0 1115825 178521 0 0 280797 0 0 41 7332238 0 948 1472379 4742 0 123236 789 0 146 936 0 36300 0 0 0 531747 0 0 1 384 213 186 498725 0 0 0 0 0 0 472 7929603 0 0 0 198 193 0 0 0 0 0 0 0 0 0 0 0 0 0
 ```
 
+### Updating ZFS and Centos
+If you need to just update the ZFS version, then a simple command like `yum upgrade zfs`. I would check with someone more knowledgeable first before you do this(Richard/Mike/Dean). In this section, I will discuss how to update a ZFS server that is in production, and what are the best practices to update both the Centos OS and the ZFS version running on it.
```

Comment:
`then a simple command like yum upgrade zfs will do` is what you meant right?
And this is for after SSHing into the zfs server I assume?

Reply:
Addressed with commit 0ab2a13